env.d/development.dist

```diff
@@ -0,0 +1,26 @@
+# Django
+DJANGO_SETTINGS_MODULE=marsha.settings
+DJANGO_CONFIGURATION=Development
+DJANGO_SECRET_KEY=ThisIsAnExampleKeyForDevPurposeOnly
+DJANGO_JWT_SIGNING_KEY=ThisIsAnExampleKeyForDevPurposeOnly
+DJANGO_BYPASS_LTI_VERIFICATION=True
+
+# Database
+POSTGRES_HOST=db
+POSTGRES_DB=marsha
+POSTGRES_USER=marsha_user
+POSTGRES_PASSWORD=pass
+
+# Python
+PYTHONUNBUFFERED=1
+
+#  AWS Admin role for development purposes
+DJANGO_AWS_ACCESS_KEY_ID=yourAwsAccesKey
+DJANGO_AWS_SECRET_ACCESS_KEY=YourAwsSecretAccessKey
+DJANGO_AWS_DEFAULT_REGION=eu-west-1
+DJANGO_AWS_SOURCE_BUCKET_NAME="dev-marsha-source"
+
+DJANGO_CLOUDFRONT_URL=
+DJANGO_CLOUDFRONT_ACCESS_KEY_ID=YourCloudFrontAccessKey
```

Comment:
> AWS Cloudfront access key ID from the master account associated with the private key at `DJANGO_CLOUDFRONT_PRIVATE_KEY_PATH`
Used to sign URLs (IAM accounts cannot be used to sign URLs)

Reply:
Note: only useful when using signed URLs. I think we can make a separate category with a title like "Cloudfront Signed URLs for mp4 files" and under it, this one, `CLOUDFRONT_PRIVATE_KEY_PATH` and `CLOUDFRONT_SIGNED_URLS_ACTIVE` which could incidentally be made an environment variable for consistency